cmd/kola/options.go

```diff
@@ -72,7 +72,7 @@ func init() {
 	if defaultRegion == "" {
 		defaultRegion = "us-west-2"
 	}
-	// Container Linux 1430.0.0 (alpha) on us-west-2
+	sv(&kola.AWSOptions.CredentialsFile, "aws-config-file", "", "AWS config file (default \"~/.aws/credentials\")")
```

Comment:
The `aws` tool distinguishes between config and credentials files, and we want the latter.  We should probably use their terminology here.

Reply:
Update pushed addressing this.